src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/Reliability/SqlConfigurableRetryLogicManager.NetCoreApp.cs

```diff
@@ -21,6 +22,9 @@ internal sealed partial class SqlConfigurableRetryLogicLoader
         /// and its related assemblies in default assembly load context if they aren't loaded yet.
         /// </summary>
         /// <returns>Resolved type if it could resolve the type; otherwise, the `SqlConfigurableRetryFactory` type.</returns>
+#if NET6_0_OR_GREATER
+        [RequiresUnreferencedCode("The type might be removed")]
```

Comment:
Removed by who? is this something that the trimmer might remove or that we, the library authors, should remove?

Reply:
![image](https://user-images.githubusercontent.com/4257079/226749687-09d42fd6-e60e-4d0e-8682-162a226ddc91.png)

Also this button allow you to directly change my bad English.